django_tables2/views.py

```diff
@@ -3,6 +3,9 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.views.generic.list import ListView
+from django.views.generic.list import (
+    MultipleObjectMixin as ConfoundingMultipleObjectMixin,
```

Comment:
What is the reason for the alias? Can't we just do `from django.views.generic.list import ListView, MultipleObjectMixin`?